src/libraries/System.IO.Compression/src/System/IO/Compression/GZipStream.cs

```diff
@@ -127,12 +300,20 @@ public override void Write(ReadOnlySpan<byte> buffer)
             }
         }
 
+        /// <summary>Reads the bytes from the current GZip stream and writes them to another stream, using a specified buffer size.</summary>
+        /// <param name="destination">The stream to which the contents of the current GZip stream will be copied.</param>
+        /// <param name="bufferSize">The size of the buffer. This value must be greater than zero. The default size is 81920.</param>
+        /// <remarks>Copying begins at the current position in the current GZip stream, and does not reset the position of the destination stream after the copy operation is complete.</remarks>
         public override void CopyTo(Stream destination, int bufferSize)
         {
             CheckDeflateStream();
             _deflateStream.CopyTo(destination, bufferSize);
         }
 
+        /// <summary>Releases the unmanaged resources used by the <see cref="System.IO.Compression.GZipStream" /> and optionally releases the managed resources.</summary>
+        /// <param name="disposing"><see langword="true" /> to release both managed and unmanaged resources; <see langword="false" /> to release only unmanaged resources.</param>
+        /// <remarks>This method is called by the public <see langword="System.ComponentModel.Component.Dispose" /> method and the <see cref="object.Finalize" /> method. <see langword="System.ComponentModel.Component.Dispose" /> invokes the protected <see cref="System.IO.Compression.GZipStream.Dispose" /> method with the <paramref name="disposing" /> parameter set to <see langword="true" />. <see cref="object.Finalize" /> invokes <see cref="System.IO.Compression.GZipStream.Dispose" /> with <paramref name="disposing" /> set to <see langword="false" />.
```

Comment:
Shouldn't it be `<see cref="..." />` instead of `<see langword="..." />`?

Reply:
You're correct; good catch @Jozkee!

```suggestion
        /// <remarks>This method is called by the public <see cref="System.ComponentModel.Component.Dispose" /> method and the <see cref="object.Finalize" /> method. <see cref="System.ComponentModel.Component.Dispose" /> invokes the protected <see cref="System.IO.Compression.GZipStream.Dispose" /> method with the <paramref name="disposing" /> parameter set to <see langword="true" />. <see cref="object.Finalize" /> invokes <see cref="System.IO.Compression.GZipStream.Dispose" /> with <paramref name="disposing" /> set to <see langword="false" />.
```